Unbundle assorted lexing/parsing/transpiling tools written in Python.
Notably NOT including jinja2 due to https://bugs.chromium.org/p/chromium/issues/detail?id=1194274
Also excluding markupsafe, because jinja2 needs an ancient version. “ImportError: cannot import name 'soft_unicode' from 'markupsafe' (/usr/lib64/python3.10/site-packages/markupsafe/__init__.py)”

--- a/mojo/public/tools/mojom/mojom/BUILD.gn
+++ b/mojo/public/tools/mojom/mojom/BUILD.gn
@@ -21,7 +21,6 @@
 
     # Third-party module dependencies
     "//third_party/jinja2/",
-    "//third_party/ply/",
   ]
 }
 
--- a/third_party/blink/renderer/bindings/scripts/build_web_idl_database.pydeps
+++ b/third_party/blink/renderer/bindings/scripts/build_web_idl_database.pydeps
@@ -1,9 +1,5 @@
 # Generated by running:
 #   build/print_python_deps.py --root third_party/blink/renderer/bindings/scripts --output third_party/blink/renderer/bindings/scripts/build_web_idl_database.pydeps third_party/blink/renderer/bindings/scripts/build_web_idl_database.py
-../../../../pyjson5/src/json5/__init__.py
-../../../../pyjson5/src/json5/lib.py
-../../../../pyjson5/src/json5/parser.py
-../../../../pyjson5/src/json5/version.py
 ../../build/scripts/blinkbuild/__init__.py
 ../../build/scripts/blinkbuild/name_style_converter.py
 build_web_idl_database.py
--- a/third_party/blink/renderer/bindings/scripts/collect_idl_files.pydeps
+++ b/third_party/blink/renderer/bindings/scripts/collect_idl_files.pydeps
@@ -4,13 +4,6 @@
 ../../../../../tools/idl_parser/idl_lexer.py
 ../../../../../tools/idl_parser/idl_node.py
 ../../../../../tools/idl_parser/idl_parser.py
-../../../../ply/__init__.py
-../../../../ply/lex.py
-../../../../ply/yacc.py
-../../../../pyjson5/src/json5/__init__.py
-../../../../pyjson5/src/json5/lib.py
-../../../../pyjson5/src/json5/parser.py
-../../../../pyjson5/src/json5/version.py
 ../../build/scripts/blinkbuild/__init__.py
 ../../build/scripts/blinkbuild/name_style_converter.py
 collect_idl_files.py
--- a/third_party/blink/renderer/bindings/scripts/generate_bindings.pydeps
+++ b/third_party/blink/renderer/bindings/scripts/generate_bindings.pydeps
@@ -1,28 +1,8 @@
 # Generated by running:
 #   build/print_python_deps.py --root third_party/blink/renderer/bindings/scripts --output third_party/blink/renderer/bindings/scripts/generate_bindings.pydeps third_party/blink/renderer/bindings/scripts/generate_bindings.py
-../../../../mako/mako/mako/__init__.py
-../../../../mako/mako/mako/_ast_util.py
-../../../../mako/mako/mako/ast.py
-../../../../mako/mako/mako/cache.py
-../../../../mako/mako/mako/codegen.py
-../../../../mako/mako/mako/compat.py
-../../../../mako/mako/mako/exceptions.py
-../../../../mako/mako/mako/ext/__init__.py
-../../../../mako/mako/mako/filters.py
-../../../../mako/mako/mako/lexer.py
-../../../../mako/mako/mako/parsetree.py
-../../../../mako/mako/mako/pygen.py
-../../../../mako/mako/mako/pyparser.py
-../../../../mako/mako/mako/runtime.py
-../../../../mako/mako/mako/template.py
-../../../../mako/mako/mako/util.py
 ../../../../markupsafe/__init__.py
 ../../../../markupsafe/_compat.py
 ../../../../markupsafe/_native.py
-../../../../pyjson5/src/json5/__init__.py
-../../../../pyjson5/src/json5/lib.py
-../../../../pyjson5/src/json5/parser.py
-../../../../pyjson5/src/json5/version.py
 ../../build/scripts/blinkbuild/__init__.py
 ../../build/scripts/blinkbuild/name_style_converter.py
 bind_gen/__init__.py
--- a/third_party/blink/renderer/modules/sanitizer_api/builtins/generate_attribute_lists.py
+++ b/third_party/blink/renderer/modules/sanitizer_api/builtins/generate_attribute_lists.py
@@ -4,7 +4,7 @@
 # found in the LICENSE file.
 """Generate list of attribute names known to this version of Chromium."""
 
-from pyjson5.src import json5
+import json5
 import optparse
 import sys
 
--- src/third_party/blink/renderer/bindings/scripts/check_generated_file_list.pydeps.old	2022-11-30 10:28:05.822183900 +0000
+++ src/third_party/blink/renderer/bindings/scripts/check_generated_file_list.pydeps	2022-11-30 19:51:45.562760400 +0000
@@ -1,28 +1,8 @@
 # Generated by running:
 #   build/print_python_deps.py --root third_party/blink/renderer/bindings/scripts --output third_party/blink/renderer/bindings/scripts/check_generated_file_list.pydeps third_party/blink/renderer/bindings/scripts/check_generated_file_list.py
-../../../../mako/mako/mako/__init__.py
-../../../../mako/mako/mako/_ast_util.py
-../../../../mako/mako/mako/ast.py
-../../../../mako/mako/mako/cache.py
-../../../../mako/mako/mako/codegen.py
-../../../../mako/mako/mako/compat.py
-../../../../mako/mako/mako/exceptions.py
-../../../../mako/mako/mako/ext/__init__.py
-../../../../mako/mako/mako/filters.py
-../../../../mako/mako/mako/lexer.py
-../../../../mako/mako/mako/parsetree.py
-../../../../mako/mako/mako/pygen.py
-../../../../mako/mako/mako/pyparser.py
-../../../../mako/mako/mako/runtime.py
-../../../../mako/mako/mako/template.py
-../../../../mako/mako/mako/util.py
 ../../../../markupsafe/__init__.py
 ../../../../markupsafe/_compat.py
 ../../../../markupsafe/_native.py
-../../../../pyjson5/src/json5/__init__.py
-../../../../pyjson5/src/json5/lib.py
-../../../../pyjson5/src/json5/parser.py
-../../../../pyjson5/src/json5/version.py
 ../../build/scripts/blinkbuild/__init__.py
 ../../build/scripts/blinkbuild/name_style_converter.py
 bind_gen/__init__.py
--- src/third_party/blink/renderer/bindings/scripts/validate_web_idl.pydeps.old	2022-11-30 10:28:05.822183900 +0000
+++ src/third_party/blink/renderer/bindings/scripts/validate_web_idl.pydeps	2022-11-30 20:00:16.870556700 +0000
@@ -1,9 +1,5 @@
 # Generated by running:
 #   build/print_python_deps.py --root third_party/blink/renderer/bindings/scripts --output third_party/blink/renderer/bindings/scripts/validate_web_idl.pydeps third_party/blink/renderer/bindings/scripts/validate_web_idl.py
-../../../../pyjson5/src/json5/__init__.py
-../../../../pyjson5/src/json5/lib.py
-../../../../pyjson5/src/json5/parser.py
-../../../../pyjson5/src/json5/version.py
 ../../build/scripts/blinkbuild/__init__.py
 ../../build/scripts/blinkbuild/name_style_converter.py
 validate_web_idl.py
